cdap-docs/developers-manual/source/building-blocks/datasets/index.rst

```diff
@@ -178,6 +177,11 @@ such as the format and schema, while abstracting from the actual underlying file
 - An extension of FileSets, ``PartitionedFileSets`` allow the associating of meta data (partitioning keys)
```

Comment:
I was thinking of it as "extension of functionality", rather than "of form", but "abstraction" could work. We could also say:

``PartitionedFileSets`` extend the functionality of FileSets by allowing the associating of meta data (partitioning keys)